### DISCUSSION TITLE
Initial sync failure on maps with more than 10 lookup fields

### DISCUSSION THREAD

**Discussion by princzkeljanos** [microsoft/Dual-write-automations#39] [Q&A]
Hi All,

[There is a known issue when entity has more than 10 lookup field initial sync fails](https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/data-entities/dual-write/dual-write-troubleshooting-initial-sync#initial-sync-failure-on-maps-with-more-than-10-lookup-fields). 

Do you know how to do the workaround programmatically using the API?

Thanks,
Jan

**Comment by Hax1337** [comment]
@princzkeljanos Currently there is no programatical workaround.

What you could do is create multiple configs and then change the version in each config and then run the automation multiple times.
